src/app/shared/mydspace-actions/claimed-task/sedici-approve-and-select/claimed-task-actions-approve-and-select.component.html

```diff
@@ -0,0 +1,4 @@
+<button (click)="openAdvancedClaimedTaskTab()" class="selectCollectionAction btn btn-primary"
```

Comment:
Es igual al claimed-task-actions-select-collection.component.html

Reply:
Si, esta bien porque hacen casi lo mismo pero no.  El ** claimed-task-actions-select-collection.component.html** es para la pantalla donde te aparecen opciones como aprobar editar y eso. El **claimed-task-actions-approve-and-select.component.html** es para poner en la edicion nadamas. A uno lo llama un decorator y al otro lo llamas directamente en el footer